### PR TITLE
Add reply icon to style kit

### PR DESCRIPTION
**Pull request by alexaubry**

### CODE REVIEW

**Comment by johnxnguyen** [comment]
Looks like you have some unwanted commits in your branch.

File: Wire-iOS Share Extension/SLComposeServiceViewController+Preview.swift [hunk 1]
```diff
@@ -87,6 +88,24 @@ extension SLComposeServiceViewController {
             }
         }
     }
+    
+    func appendLinkFromTextIfNeeded() -> [AttachmentType: [NSItemProvider]]? {
```
Comment (mikeger) [inline Wire-iOS Share Extension/SLComposeServiceViewController+Preview.swift]: Some random changes appeared

Comment (nicorsm) [reply Wire-iOS Share Extension/SLComposeServiceViewController+Preview.swift]: They're the changes from my Share Extension PR: 
https://github.com/wireapp/wire-ios/pull/2822

Comment (alexaubry) [reply Wire-iOS Share Extension/SLComposeServiceViewController+Preview.swift]: Ah right, the feature branch didn't include that change. Should I remove this commit or merge them still?

Comment (mikeger) [reply Wire-iOS Share Extension/SLComposeServiceViewController+Preview.swift]: I guess it's OK!

**Comment by mikeger** [comment]
@alexaubry don't forget to add the icon code 762 to the `ZetaIconType`!

**Pull request merged**